.github/workflows/release.yaml

```diff
@@ -28,9 +28,9 @@ jobs:
 
       - name: Add dependency chart repos
         run: |
-          helm repo add adfinis https://charts.adfinis.com && \
-          helm repo add secrets-store-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts && \
-          helm repo add bitnami https://charts.bitnami.com/bitnami
+          yq e '."chart-repos"[]' chart-testing/ct.yaml | awk 'BEGIN { FS = "=" } ; {print $1,$2}' | while read repo
```

Comment:
Is that `e` after `yq` a typo? Did you mean `-e`?

Reply:
No its correct. Its short for eval.

Reply:
We are using this yq: https://github.com/mikefarah/yq